.gitignore

```
@@ -85,3 +85,6 @@ tmp/
 ## Build artefacts
 /public/assets/
 /spec/examples.txt
+
+## IDE settings
+.vscode
```

If we do keep this:

```suggestion
/.vscode/
```

(ie only in the root and only if its a folder)

---

Would it be better to put `echo "/.vscode/" >> ~/.gitignore` in the local-env repo, then?

---

Yes, IMO (and set up Git to use a global ignore file).

---

I think it's probably a case for a template like we have for `.zshrc`, btw.